front/app/modules/commercial/content_builder/admin/components/CraftComponents/TwoColumn/index.tsx

```diff
@@ -22,18 +35,98 @@ const TwoColumn: UserComponent = () => {
       w="100%"
       gap="4px"
     >
-      <Box flex="1">
+      <Box
+        flex={columnLayout === '1-1' ? '1' : columnLayout === '1-2' ? '1' : '2'}
+      >
```

Comment:
@IvaKop I was a bit unsure about this conditional flex styling (a bit long maybe?)

Reply:
You can simply it significantly by checking only for the option that would need to be `2`
```
flex={columnLayout === '2-1' ? '2' :'1'}